src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Constants/RequestTracingConstants.cs

```diff
@@ -30,12 +30,15 @@ internal class RequestTracingConstants
         public const string KeyVaultConfiguredTag = "UsesKeyVault";
         public const string KeyVaultRefreshConfiguredTag = "RefreshesKeyVault";
         public const string ReplicaCountKey = "ReplicaCount";
+        public const string SignalrUsedTag = "UsesSignalR";
```

Comment:
Should the r be capitalized or no? Because the casing is different between the two variables.

Reply:
It should be "UserSignalr" here to keep the consistency. I will update it.

Reply:
It's alright now, but I noticed everything except the `SignalRAssemblyName` variable has a lowercase 'r', is there a reason why?

Reply:
`SignalR` with upper case R is right which stands for Realtime, not part of the word `Signal`

Reply:
For pascal casing, should it be `SignalRUserdTag` or `SignalrUserdTag`?
We used `AspNetCore` instead of `ASPNETCore`.

Reply:
It's SignalRXXX. example here: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/signalr/Microsoft.Azure.Management.SignalR/src/Generated/SignalRManagementClient.cs

Reply:
Updated to SignalR